### DISCUSSION TITLE
Create-label

### DISCUSSION THREAD

**Discussion by ShamanJon** [GAM-team/got-your-back#439] [Q&A]
For the life of me, I can't figure out how to use this action. I assume you use it to create a label and would put the name of that label after the command but it doesn't accept anything I add after the command. What am I missing?

**Comment by jay0lee** [comment]
Add

--label-restored "your label"

**Reply by ShamanJon** [reply]
Thank you for the reply. Label-restore is what I've been using to create labels to restore to during a restore but never used the create-label with it. I assumed create-label was used to just create a label within a mailbox and label-restored was for restoring to a label.
